sigstore/_cli.py

```diff
@@ -340,6 +349,15 @@ def _parser() -> argparse.ArgumentParser:
             "multiple input files"
         ),
     )
+    output_options.add_argument(
```

Comment:
My instinct would be to make this take a path

(But totally up to you!)

Reply:
Yeah, I need to think a bit about this -- `sigstore sign` can take multiple inputs, so `sigstore sign foo.txt bar.txt --bundle foo.txt` doesn't fit quite right :slightly_smiling_face: 

I'll probably do a variant of what I do for the other output flags: `--bundle` will actually enable the bundle output format, and then maybe `--sigstore-bundle <foo>` will actually write to `<fOo>` if exactly one input is provided. But that isn't great for UX either :upside_down_face:

Reply:
Ahh interesting...yeah, hard to find anything that makes sense here.

I wonder whether you need two verbs, a `sign-one` and a `sign-many`.

Happy to let it stand for now

Reply:
Hmm, here's another option: instead of allowing users to opt into Sigstore bundles with `--bundle`, maybe we enable it by default and require them to disable it with `--no-bundle`? This would come at the cost of slightly confusing files created by default (we'd essentially dupe the verification materials on disk), but we'd also end up getting the format upgrade for "free" since users wouldn't have to remember to update their CLI. 

`--bundle` itself could then be `--bundle OUTPUT`, similar to the existing options for that.

Thoughts @di?

Reply:
(This would also maybe make deprecating the existing outputs a little easier: we could begin warning that `{input}.{crt,sig,rekor}` are deprecated, and then hide them behind a `--legacy-outputs` or similar flag in an upcoming major release.

Reply:
From discussion with @tetsuo-cpp: I'm convinced that `--no-bundle` is a better route here, and that we should emit the bundle by default. From there, `--bundle` can have the same behavior as the other single-output options do (i.e. write to the given file, rather than `{input}.bundle`).

Reply:
I have a commit that does this on the way.

Reply:
I agree with the plan to use `--no-bundle` and have bundles generated by default.